components/css/college.css

```diff
@@ -0,0 +1,3963 @@
+      .collegeContent .grid .tiles .row .row-sm-3 {
+        height: 75%; }
+      .collegeContent .grid .tiles .row .row-sm-4 {
+        height: 100%; }
+      .collegeContent .grid .tiles .row .left-sm-a {
+        left: 0%; }
+      .collegeContent .grid .tiles .row .left-sm-b {
+        left: 25%; }
+      .collegeContent .grid .tiles .row .left-sm-c {
+        left: 50%; }
+      .collegeContent .grid .tiles .row .left-sm-d {
+        left: 75%; }
+      .collegeContent .grid .tiles .row .top-sm-1 {
+        top: 0%; }
+      .collegeContent .grid .tiles .row .top-sm-2 {
+        top: 25%; }
+      .collegeContent .grid .tiles .row .top-sm-3 {
+        top: 50%; }
+      .collegeContent .grid .tiles .row .top-sm-4 {
+        top: 75%; } }
+@media (max-width: 767px) {
+  .collegeContent {
+    padding: 20px 0; }
+    .collegeContent .grid .tiles {
+      padding: 20px 10px 0; }
+      .collegeContent .grid .tiles .row .row-xs-1 {
+        height: 7.22222em; }
+      .collegeContent .grid .tiles .row .row-xs-2 {
+        height: 14.44444em; }
+      .collegeContent .grid .tiles .row .row-xs-3 {
+        height: 21.66667em; }
+      .collegeContent .grid .tiles .row .row-xs-4 {
+        height: 28.88889em; } }
+/*-------------
+end tile sizing and positioning
+-------------*/
+.collegeContent .grid .introContainer {
+  padding: 10px; }
+  .collegeContent .grid .introContainer .intro {
+    width: 100%;
+    margin: -10px 0;
+    padding: 30px;
+    background-color: #d8d8d8; }
+    .collegeContent .grid .introContainer .intro h1 {
+      margin-top: 0;
+      padding-top: 0; }
+    @media (min-width: 768px) {
+      .collegeContent .grid .introContainer .intro h1 {
+        font-size: 1.75em;
+        line-height: 1.07143em; } }
+    @media (max-width: 767px) {
+      .collegeContent .grid .introContainer .intro h1 {
+        font-size: 1.55556em;
+        line-height: 1.07143em; } }
+    .collegeContent .grid .introContainer .intro a.featureLink {
+      display: block;
+      width: 100%;
+      margin: 15px 0 0;
+      padding: 7px 10px;
+      border: solid 2px #bbb;
+      color: #000; }
+    .collegeContent .grid .introContainer .intro a.featureLink:link {
+      background-color: #d8d8d8;
+      font-weight: normal;
+      text-decoration: none; }
+    .collegeContent .grid .introContainer .intro a.featureLink:visited {
+      background-color: #d8d8d8;
+      font-weight: normal;
+      text-decoration: none; }
+    .collegeContent .grid .introContainer .intro a.featureLink:hover {
+      background-color: #bbb;
+      font-weight: normal;
+      text-decoration: none; }
+    .collegeContent .grid .introContainer .intro a.featureLink:active {
+      background-color: #bbb;
+      font-weight: normal;
+      text-decoration: none; }
+.collegeContent .grid .tiles .row .tile a {
+  font-family: 'archivoNarrow',"Helvetica Neue",Helvetica,Arial,sans-serif;
+  background-color: #000;
+  color: #fff; }
+.collegeContent .grid .tiles .row .tile
```

Comment:
I just realized that there are development builds of the CSS in the components folder. There should only be SCSS in the components folder. Can we get these builds deleted?

Reply:
@jacobvenable this might be a silly question, but do you mean delete the "college.css" in the components folder or a section of code in the file?

Reply:
Sorry, delete the whole college.css. That is a build of the college.scss file and should only be located in the builds folder.